modules/querier/querier.go

```diff
@@ -876,19 +871,10 @@ func (q *Querier) SearchBlock(ctx context.Context, req *tempopb.SearchBlockReque
 }
 
 func (q *Querier) internalTagsSearchBlockV2(ctx context.Context, req *tempopb.SearchTagsBlockRequest) (*tempopb.SearchTagsV2Response, error) {
-	// check if it's the special intrinsic scope
-	// note that every block search passes the same values up. this could be handled in the frontend and be far more efficient
+	// For the intrinsic scope there is nothing to do in the querier,
```

Comment:
let's add a TODO that we can optimize this by never sending intrinsic only requests to querier and ingesters, and answer them directly from frontend